### PR TITLE
Increase timeouts over default

### PR DESCRIPTION
**Pull request by tasleson**
Depends on: https://github.com/stratis-storage/dbus-python-client-gen/pull/6

Signed-off-by: Tony Asleson <tasleson@redhat.com>

### CODE REVIEW

**Comment by tasleson** [comment]
@mulkieran Looks like we are pulling in the old version of `dbus-python-client-gen`, I'm guessing we need to bump version in it and update version requirement?

**Comment by mulkieran** [comment]
I can cut a new version of dbus-python-client-gen and post it on PyPi.

**Comment by tasleson** [comment]
@mulkieran 👍

**Comment by mulkieran** [comment]
Should be on PyPi now.

File: src/stratisd_client_dbus/_implementation.py [hunk 1]
```diff
@@ -22,16 +22,20 @@
 
 from ._data import SPECS
 
+TIME_OUT = 120  # In seconds
+
 ObjectManager = make_class(
    "ObjectManager",
-   ET.fromstring(SPECS['org.freedesktop.DBus.ObjectManager'])
+   ET.fromstring(SPECS['org.freedesktop.DBus.ObjectManager']), TIME_OUT
```
Comment (mulkieran) [inline src/stratisd_client_dbus/_implementation.py]: Please put ```TIME_OUT``` on new line, indented three spaces for this and all the rest.

Comment (tasleson) [reply src/stratisd_client_dbus/_implementation.py]: Done, although I'm not sure why 3 spaces instead of 4 like the rest of the code in the project...

Comment (mulkieran) [reply src/stratisd_client_dbus/_implementation.py]: There is a consistent rule. If the indentation is semantic, it gets four spaces, if not, three.

**Pull request merged**